content/blog/2019-09-25-c-implementation.md

```diff
@@ -0,0 +1,189 @@
++++
+title = "C Implementation of Bril"
+extra.author = "Bhargava S. Manja"
+extra.bio = """
+  [Bhargava S. Manja](https://github.com/bhargee) is a first year PhD student
+interested in computer vision, fast computing, and domain specific languages
+"""
++++
+
+## Project Report 1: C Implementation of Bril
+
+While bril is a very useful testbed for exploring existing language
+technologies and experimenting with new ideas, I was frustrated with the
+tooling around the language. It took me three or four hours of fiddling around
+with nodejs, npm, and python to get `brili` working, and I did not get either
+`bril2json` or `bril2txt` to work on my machine (I reimplemented them myself in
+python). I've rarely had such issues with my trusted systems programming
+language, C. I decided to implement a simple, fast, and correct interpreter for
+bril. I call it `cril`.
```

Comment:
This is a barnburner of an intro paragraph! I would be interested to find out, someday, why these tools were so tricky for you to get working. (And in particular why writing your own parser was easier than installing a dependency…)